trustpinning/certs.go

```diff
@@ -161,6 +161,7 @@ func ValidateRoot(prevRoot *data.SignedRoot, root *data.Signed, gun string, trus
 		logrus.Debugf("failed to verify TUF data for: %s, %v", gun, err)
 		return nil, &ErrValidationFail{Reason: "failed to validate integrity of roots"}
 	}
+	signedRoot.Signatures = root.Signatures
```

Comment:
this case is a little more tricky, since we've also validated against the old root and probably don't want to include the `isValid` markings from that check.

I had taken a stab at this in this commit (in the trust-pinning root rotation PR): https://github.com/docker/notary/pull/800/commits/a05717f80b6ceb243a19afa9b74e7ca35acab8c6